catalog/dags/providers/provider_api_scripts/stocksnap.py

```diff
@@ -166,8 +165,8 @@ def _get_metadata(item):
         return metadata
 
     @staticmethod
-    def _get_tags(item):
-        return item.get("keywords")
+    def _get_tags(item: dict) -> set:
+        return {keyword for keyword in item.get("keywords")}
```

Comment:
Is there a special reason for `keyword for keyword` here, can't we use `set(item.get("keywords", []))`? It's the same in a provider script above.

Reply:
Right, changed!